compiler/fir/analysis-tests/testData/resolve/extendedCheckers/SafeInitializationChecker/defaultArgument.kt

```diff
@@ -0,0 +1,5 @@
+class A {
+    val def = Def("Hello")
+}
+
+data class Def(<!ACCESS_TO_UNINITIALIZED_VALUE!>val a: String = run { y }<!>,  <!ACCESS_TO_UNINITIALIZED_VALUE!>val y: String = "Tmp"<!>)
```

Comment:
Why do we have two diagnostics here? IMO diagnostic on `a` should be enough

Reply:
Yes, it's true. This is my mistake. The prototype doesn't support default arguments yet, so I overlooked this error.

I note that the warning will be attached to the `y` property. The reasons for this are described below.